gatsby-config.js

```diff
@@ -61,6 +61,18 @@ module.exports = {
       resolve: `gatsby-transformer-remark`,
       options: {
         plugins: [
+          `gatsby-remark-autolink-headers`,
+          {
+            resolve: `gatsby-remark-table-of-contents`,
+            options: {
+              exclude: "Table of Contents",
+              tight: false,
+              ordered: false,
+              fromHeading: 1,
+              toHeading: 6,
+              className: "table-of-contents"
```

Comment:
I'd consider setting `elements` to just `h1`. Maybe `h2` as well if you think it's important to have links to the specific findings as well as the risk sections, but I would definitely exclude `h3` and above.